mlrun/db/base.py

```diff
@@ -776,6 +776,7 @@ def get_builder_status(
         self,
         func: "mlrun.runtimes.BaseRuntime",
         offset: int = 0,
+        events_offset: int = 0,
```

Comment:
move to last argument, in case there are calls with ordered args instead of named args.